data/remnant missions.txt

```diff
@@ -1031,8 +1031,8 @@ mission "Remnant: Return the Samples 2"
 		has "Remnant: Return the Samples: done"
 	on offer
 		conversation
-			`Following Plume's directions, you find a lab tucked behind the main area of the spaceport, where a short walkway leads into an underground structure. Stepping inside the door you note a rack of camouflage netting positioned easily at hand. Given what you have learned of their history, it is probably a last ditch preparation in case they need to hide the entrance from a raid.`
-			`	The lab itself is a microcosm of Remnant development, the structure appears to be some strange semi-organic material, and you can see an eclectic mix of familiar and unfamiliar shapes cluttering the tables. If you had to guess, you would think that this lab was the result of several different families of technology, only one of which was human.`
+			`Following Plume's directions, you find a lab tucked behind the main area of the spaceport, where a short walkway leads into an underground structure. Stepping inside the door you note a rack of camouflage netting positioned easily at hand. Given what you have learned of their history, it is probably a last-ditch preparation in case they need to hide the entrance from a raid.`
+			`	The lab itself is a microcosm of Remnant development. The structure appears to be some strange semi-organic material, and you can see an eclectic mix of familiar and unfamiliar shapes cluttering the tables. If you had to guess, you would say that this lab was the result of several different families of technology, only one of which was human.`
```

Comment:
I'm fine with think tbh, but no hard feelings either way

Reply:
The dash between last and ditch makes sense.

The comma after development, though... I think that might actually be better as a colon.

As to changing the think to say, I disagree with. I don't think that's really a necessary change.

Reply:
Full stop or colon, either way... just not a comma!

say → think: To elaborate on the curt commit message, my reasoning was that this text is describing your internal monologue. So, of course you _think_ that. Also, this may just be my idiom, but "if [I] had to guess, [I] would think" seemed awkward to me. It may be the implication of "I will make a guess, but I'll only think it, not say it" that strikes me as odd.

Reply:
Agreed on that. The comma needs to go. Switch that to a colon and that will be good.

Sounds like the say/think part will need more input from the community. Maybe leave it off this PR and/or get more feedback on it before including it.

Reply:
I'd say that both say and think work here, and one doesn't necessarily sound "more correct" to me than the other. When a situation like this pops up (i.e. the change isn't agreed upon as being an improvement, but it isn't wrong either), we usually air on the side of just leaving it as it was.